target/linux/ath79/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -109,6 +109,10 @@ case "$FIRMWARE" in
 		ath10kcal_extract "art" 20480 2116
 		ath10kcal_patch_mac $(macaddr_add $(cat /sys/class/net/eth1/address) -1)
 		;;
+	tplink,archer-c7-v5)
```

Comment:
Nothing relevant for you, but this seems to be incorrect for the Archer A7 v5 as qca9563 does only have 1 Ethernet MAC and 5GHz WiFi MAC is derived from eth1.